guides/common/attributes-satellite.adoc

```diff
@@ -89,26 +89,23 @@
 :Team: Red{nbsp}Hat
 
 
-//For 7.0 GA
 // Repositories and subscriptions (used in Satellite guides)
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-capsule-{ProductVersion}-rpms
-:RepoRHEL8ServerSatelliteServerProductVersion: satellite-{ProductVersion}-for-rhel-8-x86_64-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.10-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-{ProductVersion}-rpms
-:RepoRHEL8ServerSatelliteCapsuleProductVersion: satellite-capsule-{ProductVersion}-for-rhel-8-x86_64-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-{ProductVersion}-rpms
-:RepoRHEL8ServerSatelliteMaintenanceProductVersion: satellite-maintenance-{ProductVersion}-for-rhel-8-x86_64-rpms
+
+// For beta
+:RepoSatelliteVersion: 7-beta
+:BaseServerRepoID: server-7
+// For a release
+:RepoSatelliteVersion: {ProductVersion}
+:BaseServerRepoID: 7-server
```

Comment:
If you're going to use this, shouldn't this be
```suggestion
:BaseServerRepoID: rhel-7-server
```

Reply:
@ekohl, maybe yes, but I dropped this attribute in the last commit. After my alert related to the inconsistency of naming Beta and GA repos, it has been unified, so an attribute to distinguish between Beta and GA is no longer needed.

Reply:
And that's why it's great to get this work going sooner rather than later. Now it was still possible to fix. Glad it got fixed, makes everyone's lives easier.

Reply:
Definitely :-)